modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/utils/types.ts

```diff
@@ -78,6 +77,12 @@ export interface IEmptyStateConfiguration extends IEmptyState {
 	};
 }
 
+export enum EStateInURLSettings {
+	OFF = 'off',
+	PUSH = 'push',
+	REPLACE = 'replace',
+}
```

Comment:
To be more descriptive, maybethis could be `EStateInURLBehavior`, with values something like
```js
EStateInURLBehavior.STORE_ONLY_LAST_STATE,
EStateInURLBehavior.STORE_STATE_HISTORY,
EStateInURLBehavior.OFF.
```